database/migrations/2021_03_02_100000_create_states_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStatesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('states', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('slug')->unique()->index();
+            $table->string('title')->unique();
+            $table->string('abbreviation', 2)->unique();
+            $table->string('description')->nullable();
+            $table->string('capital')->nullable();
+            $table->foreignIdFor(app('country'));
```

Comment:
Let's just use the classes for now. I need to update the config file in tipoff/support to register these new classes in the container.